typings/jsonwebtoken.d.ts

```diff
@@ -0,0 +1,60 @@
+declare module 'jsonwebtoken' {
```

Comment:
could just install this instead https://www.npmjs.com/package/@types/jsonwebtoken

Reply:
Can't install that.  That one of the typings files I was talking about being incorrect.  The returns types for the `decode` and `verify` functions are incorrect.

Reply:
I could include a comment at the top of `typings/jsonwebtoken.d.ts` explaining why we need it?

Reply:
ok, no worries then